app/javascript/shared/components/__tests__/tags.test.jsx

```diff
@@ -4,11 +4,14 @@ import { render, fireEvent } from '@testing-library/preact';
 import { axe } from 'jest-axe';
 import { Tags } from '../tags';
 
+fetch.enableMocks();
```

Comment:
Instead of mocking fetch, I'm going to start a conversation about moving us towards Mock Service Worker (msw), https://kentcdodds.com/blog/stop-mocking-fetch, for front-end unit/component tests.